### PR TITLE
Added commits to GitHubIssue

### PR DESCRIPTION
**Pull request by alankstewart**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <java.version>1.8</java.version>
         <scala.version>2.11.8</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
-        <rug.version>(0.25.0,1.0.0-m.5]</rug.version>
+        <rug.version>(0.25.0,1.0.0-m.6]</rug.version>
```
Comment (ddgenome) [inline pom.xml]: Why wouldn't this be `[1.0.0-m.6,2.0.0)`?

Comment (alankstewart) [inline pom.xml]: You're probably right. This range scheme came before I looked at this lib and I'm not sure why it is like this. I'll update it

**Comment by ddgenome** [comment]
@alankstewart do you want to proceed with this or wait until the model supports issue-commit relationships?

**Comment by alankstewart** [comment]
Perhaps it can be merged now and I can easily remove the issue events and getCommit part once the model is updated

**Pull request merged**